docs/en/query_language/syntax.md

```diff
@@ -99,15 +101,52 @@ For more information, see the section "Operators" below.
 
 Data types and table engines in the `CREATE` query are written the same way as identifiers or functions. In other words, they may or may not contain an arguments list in brackets. For more information, see the sections "Data types," "Table engines," and "CREATE".
 
-## Synonyms
+## Expression aliases
 
-In the SELECT query, expressions can specify synonyms using the AS keyword. Any expression is placed to the left of AS. The identifier name for the synonym is placed to the right of AS. As opposed to standard SQL, synonyms are not only declared on the top level of expressions:
+Alias is a developer defined name for expression in a query.
 
-``` sql
-SELECT (1 AS n) + 2, n
+```
+expr AS alias
+```
+
+- `AS` — keyword for defining aliases. You can define alias for table or a column name skipping `AS` keyword.
+
+    For example, `SELECT b.column_name from t b`.
+
+- `expr` — any expression supported by ClickHouse.
+
+    For example, `SELECT (1 AS n) + 2, n`.
```

Comment:
This example shows unusual feature that's specific to ClickHouse, normally `AS` can be written only externally (after `2` in this example). So I'd start with more simple example and explain this separately.